packages/interface/src/AppRouter.tsx

```diff
@@ -13,23 +13,23 @@ type LazyComponent = Promise<{
 
 // I'm still so annoyed at this lol
 const lazily = (loader: () => unknown) => React.lazy(() => loader() as LazyComponent)
-const Home = lazily(() => import('./pages/Home'))
-const LibraryOverview = lazily(() => import('./pages/library/LibraryOverview'))
-const LibraryFileExplorer = lazily(() => import('./pages/library/LibraryFileExplorer'))
-const CreateLibrary = lazily(() => import('./pages/library/CreateLibrary'))
-const SeriesOverview = lazily(() => import('./pages/SeriesOverview'))
-const BookOverview = lazily(() => import('./pages/book/BookOverview'))
-const ReadBook = lazily(() => import('./pages/book/ReadBook'))
-const ReadEpub = lazily(() => import('./pages/book/ReadEpub'))
-const SettingsLayout = lazily(() => import('./components/settings/SettingsLayout'))
-const GeneralSettings = lazily(() => import('./pages/settings/GeneralSettings'))
-const JobSettings = lazily(() => import('./pages/settings/JobSettings'))
-const ServerSettings = lazily(() => import('./pages/settings/ServerSettings'))
-const UserSettings = lazily(() => import('./pages/settings/UserSettings'))
-const FourOhFour = lazily(() => import('./pages/FourOhFour'))
-const ServerConnectionError = lazily(() => import('./pages/ServerConnectionError'))
-const LoginOrClaim = lazily(() => import('./pages/LoginOrClaim'))
-const OnBoarding = lazily(() => import('./pages/OnBoarding'))
+const Home = lazily(() => import('./pages/Home.js'))
```

Comment:
> Fixed some tedious lint errors in AppRouter

I assume adding `.js` was part of this? What was the lint?

Reply:
Yes it was, also for some reason anything but 'js' would trigger an error as well.

Can't remember what the linter was, I'll let you know in the morning it's 11:30pm here and I'm done with code for the day 😅

Reply:
Huh, interesting - alright. I was worried those changes wouldn't bundle properly since those files with `.js` don't technically exist, but locally building it seems to have properly picked it up so 🤷 

> Can't remember what the linter was, I'll let you know in the morning it's 11:30pm here and I'm done with code for the day 😅

Haha no worries, have a good night!